dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/values/AccessLogRemoteHost.java

```diff
@@ -49,7 +49,7 @@ public static String getRemoteHostAddress(HttpResponseMessage response, HttpRequ
 
             if (Objects.nonNull(serviceContext)) {
 
-                hostAddress = serviceContext.useForwardedHeadersInAccessLog() ? serviceContext.getForwardedRemoteHost() : null;
+                hostAddress = serviceContext.useRemoteIpOptionsInAccessLog() ? serviceContext.getForwardedRemoteHost() : null;
```

Comment:
I'm unsure that this is more descriptive since the object that will be used is getForwardedRemoteHost() . While it is true that the remote host is calculated based on remote IP options, it itself is a forwarded (or x-forwarded) type header. So as it stands it would read : "use forwarded header or host address for the access log remote host" . We don't configure a remoteIP option that gives us a host, rather those options help determine if we consider the forwarded headers. So I'd lean towards leaving at least this one as is.